doc/user/acl.md

```diff
@@ -0,0 +1,150 @@
+# DAOS Access Control
+
+Client access to DAOS resources like pools and containers is controlled by
+Access Control Lists (ACLs). A DAOS ACL is a list of zero or more Access Control
+Entries (ACEs). ACEs are the individual rules applied to each access decision.
+
+## Access Control Entries
+
+In the input and output of DAOS tools, an ACE is defined using a colon-separated
+string format:\
+`TYPE:FLAGS:IDENTITY:PERMISSIONS`
+
+The contents of all the fields are case-sensitive.
+
+### Type
+
+The type of entry. Only one type of ACE is supported at this time.
+
+* A (Allow): Allow access to the specified identity for the given permissions.
+
+### Flags
+
+The flags provide additional information about how the ACE should be
+interpreted.
+
+* G (Group): The identity should be interpreted as a group.
+
+### Identity
+
+The identity (also called the principal) is specified in the name@domain format.
+The domain should be left off if the name is a UNIX user/group on the local
+domain. Currently, this is the only case supported by DAOS.
+
+There are three special identities, `OWNER@`, `GROUP@` and `EVERYONE@`,
+which align with User, Group, and Other from traditional POSIX permission bits.
+When providing them in the ACE string format, they must be spelled exactly as
+written here, in uppercase with no domain appended. The `GROUP@` entry must
+also have the `G` (group) flag.
+
+### Permissions
+
+The permissions in a resource's ACL permit a certain type of user access to
+the resource. Which types of permissions are valid depends on the resource
+type.
```

Comment:
Maybe add sentence "Any permissions not specifically associated with containers apply to both pools and containers." or something like this.

Reply:
Hm, my hesitation is, that's not strictly true - create container only applies to pools, so it really goes both ways. The sentence "Which types of permissions..." feels a little redundant on a re-reading, though. Perhaps it should be removed entirely? The table conveys that there are different meanings for pool and container permissions.